src/zenml/steps/external_artifact.py

```diff
@@ -11,245 +11,16 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-"""External artifact definition."""
-import os
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Optional,
-    Type,
-    Union,
-)
-from uuid import UUID, uuid4
+"""Backward compatibility for the `ExternalArtifact` class."""
 
-from zenml.client import Client
-from zenml.io import fileio
+from zenml.artifacts.external_artifact import ExternalArtifact
 from zenml.logger import get_logger
-from zenml.materializers.base_materializer import BaseMaterializer
-from zenml.materializers.materializer_registry import materializer_registry
-from zenml.utils import artifact_utils, source_utils
-
-if TYPE_CHECKING:
-    from zenml.config.source import Source
-
-    MaterializerClassOrSource = Union[str, "Source", Type["BaseMaterializer"]]
 
 logger = get_logger(__name__)
+logger.warning(
+    "The `ExternalArtifact` object was moved to `zenml.artifacts.external_artifact`. "
+    "This import path is going to be deprecated, please consider updating your "
```

Comment:
```suggestion
    "This import path is going to be deprecated. Please consider updating your "
```

And can you update https://www.notion.so/zenml/Deprecations-903bcaacb5c04954b9f6133b3340a312 as well?